lib/Ed25519KeyPair.js

```diff
@@ -68,7 +107,20 @@ class Ed25519KeyPair extends LDKeyPair {
       ...options
     });
   }
-
+  /**
+   * @description Creates an ED25519 Key Pair from an existing private key.
+   * @example
+   * const options = {
+   *   privateKeyBase58: 'E9873D79C6D87DC0FB6A5778633389F4453213303DA61F20BD67FC233AA33262'
+   * };
+   * const key = Ed25519KeyPair.from(options);
+   * @param {Object} options - Contains a private key.
+   * @param {Object} [options.privateKey = undefined] - A private key.
+   * @param {string} [options.privateKeyBase58 = undefined] - A Base 56
+   * Private key string.
+   *
+   * @returns {Ed25519KeyPair} An Ed 25519 Key Pair.
```

Comment:
I'm not sure how redundant we need these to be.  But at least a space issue here.
```suggestion
   * @returns {Ed25519KeyPair} An Ed25519 Key Pair.
```